redata/backends/exasol.py

```diff
@@ -103,26 +103,27 @@ def check_count_per_value(self, table, checked_column, time_interval):
         ).fetchall()
         return [SimpleNamespace(**row) for row in result]
 
-    def check_data_volume(self, table, where_timecol):
+    def check_data_volume(self, table, time_interval):
+        interval_part = self.db.make_interval(time_interval)
         result = self.db.execute(
             f"""
             SELECT
                 count(*) as "count"
             FROM {table.table_name}
-            WHERE [{table.time_column}] {where_timecol}
+            WHERE [{table.time_column}] > now() - {interval_part}
             """,
             fetch_dict=True,
         ).fetchone()
         return SimpleNamespace(**result)
     
-    def check_data_volume_diff(self, table, where_timecol):
+    def check_data_volume_diff(self, table, from_time):
```

Comment:
```suggestion
    def check_data_volume_diff(self, table, from_time):
        interval_part = self.db.make_interval(time_interval)
```

Reply:
Hey, for those 2 suggestions with time_interval, I think in this function `check_data_volume_diff` there is no time_intervals, so I would keep the current version. 
I think the one having time_interval is `check_data_volume`, this one I kept untouched :).